### PR TITLE
Changes to regular shooting and intro to amp shooting

### PR DESCRIPTION
**Pull request by Sahiltheram**

### CODE REVIEW

File: src/main/java/frc/robot/commands/sequences/ExitAndShoot.java [hunk 1]
```diff
@@ -10,7 +12,11 @@ public class ExitAndShoot extends ParallelCommandGroup{
     public ExitAndShoot(Shooter shooter, Feeder feeder) {
         addCommands(
                 new ShootSpeaker(shooter),
-                new FeederGamepieceUntilLeave(feeder)
+                new SequentialCommandGroup(
```
Comment (armadilloz) [inline src/main/java/frc/robot/commands/sequences/ExitAndShoot.java]: Just add a comment on what this is meant to do.

File: src/main/java/frc/robot/constants/Constants2024.java [hunk 2]
```diff
@@ -34,9 +34,12 @@ public class Constants2024 extends GameConstants {
     public static final double SHOOTER_MOTOR_LEFT_SPEED = 0.6;
     public static final double SHOOTER_MOTOR_RIGHT_SPEED = 1;
 
+    public static final double SHOOTER_MOTOR_RIGHT_AMP_SPEED = 0.25;
```
Comment (armadilloz) [inline src/main/java/frc/robot/constants/Constants2024.java]: Add these to the 2023 constants as well, make sure to also build with the  2023 constants.

File: src/main/java/frc/robot/constants/GameConstants.java [hunk 3]
```diff
@@ -14,7 +14,7 @@ public class GameConstants {
 
     //DEBUGS
     public static final boolean SHOOTER_DEBUG = false;
-    public static final boolean RAMP_DEBUG = false;
+    public static final boolean RAMP_DEBUG = true;
```
Comment (armadilloz) [inline src/main/java/frc/robot/constants/GameConstants.java]: Bring that back to FALSE

**Review by armadilloz — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/constants/Constants2024.java (hunk 2 above)
Comment (neo-Aditya) [inline src/main/java/frc/robot/constants/Constants2024.java]: Add these to Testbed constants too

File: src/main/java/frc/robot/commands/sequences/ExitAndShoot.java (hunk 1 above)
Comment (Sahiltheram) [reply src/main/java/frc/robot/commands/sequences/ExitAndShoot.java]: Done

File: src/main/java/frc/robot/constants/Constants2024.java (hunk 2 above)
Comment (Sahiltheram) [reply src/main/java/frc/robot/constants/Constants2024.java]: Done

File: src/main/java/frc/robot/constants/GameConstants.java (hunk 3 above)
Comment (Sahiltheram) [reply src/main/java/frc/robot/constants/GameConstants.java]: done

File: src/main/java/frc/robot/constants/Constants2024.java (hunk 2 above)
Comment (Sahiltheram) [reply src/main/java/frc/robot/constants/Constants2024.java]: Done

**Review by armadilloz — APPROVED** [review]

**Review by armadilloz — APPROVED** [review]

**Pull request merged**